lib_standard_app/io.h

```diff
@@ -96,4 +119,7 @@ WEAK int io_send_response_buffer(const buffer_t *rdata, uint16_t sw);
  * @return zero or positive integer if success, -1 otherwise.
  *
  */
-WEAK int io_send_sw(uint16_t sw);
+static inline int io_send_sw(uint16_t sw)
+{
+    return io_send_response_buffer(NULL, sw);
```

Comment:
```suggestion
    return io_send_response_buffers(NULL, 0, sw);
```
Else you are going to deference NULL